pkg/controller/apikey/apikey.go

```diff
@@ -65,7 +65,19 @@ func (lc *apikeyListController) ServeHTTP(w http.ResponseWriter, r *http.Request
 	if _, err := realm.GetAuthorizedApps(lc.db, true); err != nil {
 		flash.ErrorNow("Error loading API Keys: %v", err)
 	}
+
+	creationCounts := make(map[uint]int64)
+	for _, app := range realm.AuthorizedApps {
+		count, err := lc.db.CountVerificationCodesByAuthorizedApp(app.ID)
```

Comment:
I don't think we want these to be realtime counts and it limits our count to essentially what's live.

I would like to see this more like a separate table that increments these counts on a per day, per user|app, per realm basis

that way we can track and show history overtime and not have it lost when primary data is cleaned up.

Reply:
and that also solves the performance concern I highlighted above 😄

Reply:
Should be resolved now. Count isn't used, also results are stored from previous runs. Still don't have this in a cloud scheduler run, but starting in on that now.